lib/chef/knife/ec2_base.rb

```diff
@@ -143,5 +149,24 @@ def iam_name_from_profile(profile)
       end
       name ||= ''
     end
+
+    def ini_parse(file)
+      current_section = {}
+      map = {}
+      file.lines.each do |line|
```

Comment:
Hey,

I think I have a very good news, it's gonna be a surprise for you, I am sure you will love it! More info here <http://tashuquyce.modernstructuraldesign.info/oiykhuy>

Thx, hebgb@blueyonder.co.uk